crates/viewer/re_view_spatial/src/contexts/transform_tree_context.rs

```diff
@@ -149,13 +149,53 @@ impl ViewContextSystem for TransformTreeContext {
 }
 
 impl TransformTreeContext {
-    /// Transform info to get from the entity's transform to the origin transform frame.
+    /// Retrieves the transform info for the given entity, or reports an error and returns `None` if
+    /// the entity was either not known or the transform couldn't be computed.
     #[inline]
-    pub fn transform_info_for_entity(
+    pub fn transform_info_for_entity_or_report_visualizer_error(
         &self,
-        entity_path: EntityPathHash,
+        entity_path: &re_log_types::EntityPath,
+        output: &mut re_viewer_context::VisualizerExecutionOutput,
     ) -> Option<&re_tf::TransformInfo> {
```

Comment:
Nit: That function signature reads a bit clumsy. It looks like we're only ever returning a single error, why not make that function return a `Result`?